.github/workflows/github-actions.yml

```diff
@@ -20,19 +20,18 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  python3 -m pip install pipenv
-                  pipenv install --dev
+                  python3 -m pip install -e ".[test]"
```

Comment:
You could just list out the individual packages you need specific for this action, since most aren't needed to run tests (flake8). You could also break it out into test / dev packages in the options.extras_require.